Makefile

```diff
@@ -30,4 +30,12 @@ test:
 	poetry run python -m unittest discover -s tests -t .
 
 requirements:
-	poetry export -f requirements.txt --output requirements.txt --without-hashes
\ No newline at end of file
+	poetry export -f requirements.txt --output requirements.txt --without-hashes
+
+requirements-dev:
+	poetry export -f requirements.txt --with dev --output requirements-dev.txt --without-hashes
+
+pipeline:
+	kustomize build manifests/prerequisites | kubectl apply -f -
+	kustomize build manifests/ | kubectl apply -f -
+	kubectl apply -f manifests/pipeline/numalogic-prometheus-pipeline.yaml
```

Comment:
You do not need to do pipeline apply again, line 40 covers it

Reply:
Cool. Changing